release/release.go

```diff
@@ -108,17 +110,34 @@ func readOSRelease() OS {
 	return osRelease
 }
 
+var ioutilReadFile = ioutil.ReadFile
+
+func isWSL() bool {
+	version, err := ioutilReadFile("/proc/version")
```

Comment:
Any idea whether Cygwin exposes /proc/version too? I tracked down this patch http://cygwin.1069669.n5.nabble.com/PATCH-add-information-to-proc-version-td91560.html but I'm not sure if it got applied or not.